src/main/kotlin/app/revanced/patches/youtube/general/layoutswitch/LayoutSwitchPatch.kt

```diff
@@ -20,7 +20,7 @@ import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
 @Suppress("unused")
 object LayoutSwitchPatch : BaseBytecodePatch(
     name = "Layout switch",
-    description = "Adds an option to trick dpi to use tablet or phone layout.",
+    description = "Adds an option to spoof the dpi in order to use a tablet or phone layout.",
```

Comment:
IDK if "spoof" is the correct technical term. I'll revert this change if you don't like it.